api/capi/include/nntrainer_internal.h

```diff
@@ -108,11 +109,19 @@ typedef struct {
  * @retval #ML_ERROR_INVALID_PARAMETER Invalid parameter.
  * @retval #ML_ERROR_CANNOT_ASSIGN_ADDRESS Cannot assign object.
  */
-int ml_nnmodel_get_layer(ml_nnmodel_h model, const char *layer_name,
-                         ml_nnlayer_h *layer);
+int ml_nnmodel_get_layer(ml_train_model_h model, const char *layer_name,
```

Comment:
It'd be more readable if you use a common prefix for internal functions, too; e.g., ```_ml_train_```

Reply:
done.